engine/src/main/java/org/terasology/engine/config/flexible/ui/AutoConfigWidgetFactory.java

```diff
@@ -79,7 +79,7 @@ public UIWidget buildWidgetFor(AutoConfig config) {
             Optional<UIWidget> settingWidget = settingWidgetFactory.createWidgetFor(setting);
 
             if (!settingWidget.isPresent()) {
-                logger.error("Couldn't find a widget for the Setting [{}]", setting.getHumanReadableName());
+                logger.atError().addArgument(() -> setting.getHumanReadableName()).log("Couldn't find a widget for the Setting [{}]");
```

Comment:
Simple getter method so impact neglectable
```suggestion
                logger.error("Couldn't find a widget for the Setting [{}]", setting.getHumanReadableName()); //NOPMD
```